### ISSUE TITLE
Auth issues when using API key

### ISSUE THREAD

**Issue by ljdelight** [maproulette/maproulette-backend#1129]
Requests using the API fail when using the apiKey for authN/Z. This was caused by the upgrade to Play 2.9 because the Play Application secret was also (and unknowingly) used on the backend's side for encrypting certain backend-private data.

An example API call using the apiKey header:

```sh
curl -H "apiKey:<TODO_APIKEY>" -X GET https://maproulette.org/api/v2/user/whoami | jq
```

**Comment by ljdelight** [comment]
The resolution is described in https://github.com/maproulette/maproulette-backend/pull/1128 : Play 2.9 introduced a change where its application key must be 32 bytes or longer, and this impacted MapRoulette's cryptography with its own internal items. It's easier to manage these separately, similar to how the OSM secrets are not used to encrypt data stored within MapRoulette.

**Comment by ljdelight** [comment]
This was resolved in v4.5.1 and deployed. Closing.
